lib/edge_impulse/CMakeLists.txt

```diff
@@ -59,8 +59,14 @@ ExternalProject_Add(edge_impulse_project
 # causes the Edge impulse library to be fetched on each build invocation.
 # Note: This also results in the `ALL` target to always be considered out-of-date.
 if(CONFIG_EDGE_IMPULSE_DOWNLOAD_ALWAYS)
+  if(${CMAKE_VERSION} VERSION_LESS "3.14")
+    set(REMOVE_COMMAND remove)
+  else()
+    set(REMOVE_COMMAND rm)
+  endif()
```

Comment:
This will break with CMake 3.15 or 3.16, cause the `rm` is new in CMake 3.17.

You should use: `   if(${CMAKE_VERSION} VERSION_LESS "3.17")`

Reply:
Thanks, I missed it